taxi/urls.py

```diff
@@ -18,7 +13,10 @@
     ),
     path("cars/", CarListView.as_view(), name="car-list"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
-    path("drivers/", DriverListView.as_view(), name="driver-list"),
+    path("drivers/",
+         DriverListView.as_view(),
+         name="driver-list"
```

Comment:
The path definition for the `driver-list` view is split across multiple lines. For consistency and readability, consider formatting it similarly to the other path definitions, such as `car-list` or `car-detail`, which are on a single line.